std/math.d

```diff
@@ -553,13 +553,13 @@ auto abs(Num)(Num x)
 // workaround for https://issues.dlang.org/show_bug.cgi?id=18251
 //if (!isDeprecatedComplex!Num &&
     //(is(typeof(Num.init >= 0)) && is(typeof(-Num.init)) ||
-    //(is(Num == short) || is(Num == byte))))
+    //(is(Num : const(short)) || is(Num : const(byte)))))
 {
     static if (isFloatingPoint!(Num))
         return fabs(x);
     else
     {
-        static if (is(Num == short) || is(Num == byte))
+        static if (is(Num : const(short)) || is(Num : const(byte)))
```

Comment:
Note, this isn't the same. `ubyte`, `char`, `wchar`, etc. are going to pass these.

I suggest `is(Unqual!Num == short)`

Reply:
okay, done.